activemq-kahadb-store/src/main/java/org/apache/activemq/store/kahadb/MessageDatabase.java

```diff
@@ -2236,7 +2236,7 @@ private boolean shouldForward(JournalCommand<?> command) {
                 result = true;
             } else if (command instanceof KahaCommitCommand) {
                 KahaCommitCommand kahaCommitCommand = (KahaCommitCommand) command;
-                if (kahaCommitCommand.hasTransactionInfo() && kahaCommitCommand.getTransactionInfo().hasXaTransactionId()) {
+                if (kahaCommitCommand.hasTransactionInfo())  {
```

Comment:
we don't need that check for hasTransactionInfo any more, it was redundant before but is more so now. But it is not a biggie.

Reply:
That does make sense - I guess we wouldn't see a commit command without transaction info. I'll remove that check.

Reply:
thanks. less is always more.